### PR TITLE
331 fe code snippet

### PR DESCRIPTION
**Pull request by erenfn**
- Solves #331 

- Also refactored team count in backend

image

### CODE REVIEW

File: backend/src/service/team.service.js [hunk 1]
```diff
@@ -25,45 +25,54 @@ class TeamService {
...
+            throw new Error("Get Team Count Error");
+        }
+    };
+
     async updateTeam(name) {
         try {
             await Team.update({
                 name: name
-            },{
+            }, {
```
[the remark this answers is not in the document]
Comment (uparkalau) [reply backend/src/service/team.service.js:48]: Do you want to resolove it? @erenfn

File: backend/src/service/team.service.js [hunk 2]
```diff
@@ -25,45 +25,54 @@
...
                 where: {}
             });
         }
-        catch(error) {
+        catch (error) {
             throw new Error("Error Updating Team");
         }
     }
```
[the remark this answers is not in the document]
Comment (uparkalau) [reply backend/src/service/team.service.js:55]: Since we were using transactions it make sense to implement it here .

**Review by uparkalau — CHANGES_REQUESTED** [review]
Review Code rabbit recommendation and implement what is make sense.

**Comment by erenfn** [comment]
> Review Code rabbit recommendation and implement what is make sense.

@uparkalau done

**Review by uparkalau — APPROVED** [review]

**Pull request merged**